site/content/en/docs/Overview/architecture.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Architecture"
+linkTitle: "Architecture"
+weight: 1
+description: >
+  Prow is made up of a collection of microservices (aka "Prow components") that work together in a **service cluster**, leveraging one or more **build clusters** to schedule Prow Jobs (or just "jobs").
+---
+
+## Prow in a Nutshell
+
+Prow creates **Prow Jobs** ("jobs") based on either GitHub events (e.g., a new
+PR is created, or is merged, or a person comments "/retest" on a PR). These jobs
+are created inside the Service Cluster as Kubernetes Custom Resources. The
+**Prow Controller Manager** takes pending jobs and schedules them into a build
+cluster, where they run as Kubernetes pods. **Crier** then reports back the
+results back to GitHub.
+
+Prow can also create jobs based on time; these are created by **Horologium** and
+are called **periodic** jobs.
+
+```mermaid
+flowchart TD
+
+
+    classDef yellow fill:#ff0
+    classDef cyan fill:#0ff
+    classDef pink fill:#f99
+
+    subgraph Service Cluster["<span style='font-size: 40px;'><b>Service Cluster</b></span>"]
+    Deck:::cyan
+    Prowjobs:::yellow
+    Crier:::cyan
+    Tide:::cyan
+    Hook:::cyan
+    Horologium:::cyan
+    Sinker:::cyan
+    PCM[Prow Controller Manager]:::cyan
+    PluginCat(["Hook 'cat' plugin"])
+    PluginTrigger(["Hook 'trigger' plugin"])
+    end
+
+    subgraph Build Cluster[<b>Build Cluster</b>]
+    Pods[(Pods)]:::yellow
+    end
+
+    style Legend fill:#fff,stroke:#000,stroke-width:4px
+    subgraph Legend["<span style='font-size: 20px;'><b>LEGEND</b></span>"]
+    direction LR
+    k8sResource[Kubernetes Resource]:::yellow
+    prowComponent[Prow Component]:::cyan
+    hookPlugin([Hook Plugin])
+    Other
+    end
+
+    Prowjobs <-.-> Deck -----> |Serve| prow.k8s.io
+    GitHub ==> |Webhooks| Hook
+    Prowjobs <-.-> Tide --> |Retest and Merge| GitHub
+    Horologium ---> |Create| Prowjobs
+    Hook --> |/meow| PluginCat
+    Hook --> |/retest| PluginTrigger
```

Comment:
nit: What about moving the plugins inside the "Hook" box to indicate they are part of the hook component? Doesn't matter much.

Reply:
Done